src/upgrades/mongodb_upgrade.py

```diff
@@ -112,10 +138,36 @@ def _on_force_upgrade_action(self, event: ActionEvent) -> None:
             return
         logger.debug("Forcing upgrade")
         event.log(f"Forcefully upgrading {self.unit.name}")
-        self._upgrade.upgrade_unit()
+        self._upgrade.upgrade_unit(charm=self.charm)
         event.set_results({"result": f"Forcefully upgraded {self.unit.name}"})
         logger.debug("Forced upgrade")
 
+    def post_upgrade_check(self, event: EventBase):
+        """Runs the post upgrade check to verify that the cluster is healthy.
+
+        By deferring before setting unit state to HEALTHY, the user will either:
+            1. have to wait for the unit to resolve itself.
+            2. have to run the force-upgrade action.
```

Comment:
```suggestion
            2. have to run the force-upgrade action (to upgrade the next unit).
```
nit: want to clarify that the user doesn't run force-upgrade on that unit not set to HEALTHY